erts/doc/src/erl_cmd.xml

```diff
@@ -216,6 +216,39 @@
           <seeerl marker="kernel:application">
           <c>application(3)</c></seeerl>.</p>
       </item>
+      <tag><c><![CDATA[-configfd FD [FD ...]]]></c></tag>
+      <item>
+        <p>Specifies the name of one or more file descriptors with
+        configuration data for applications;
+        see <seefile marker="kernel:app"><c>app(4)</c></seefile> and
+        <seeerl marker="kernel:application">
+        <c>application(3)</c></seeerl>. The parameter
+        <c><![CDATA[-configfd 0]]></c> implies
+        <c><![CDATA[-noinput]]></c>.</p>
+        <p>Examples (Unix shell):</p>
+<pre>
+$ <input>erl \
+-noshell \
+-configfd 3 \
+-eval \
+'io:format("~p~n",[application:get_env(kernel, logger_level)]),erlang:halt()' 3&lt; \
+&lt;(echo '[{kernel, [{logger_level, warning}]}].')</input>
+{ok,warning}
+</pre>
+<pre>
+$ <input>echo '[{kernel, [{logger_level, warning}]}].' > test1.config</input>
+$ <input>echo '[{kernel, [{logger_level, error}]}].' > test2.config</input>
+$ <input>erl \
+-noshell \
+-configfd 3 \
+-configfd 4 \
+-eval \
+'io:format("~p~n",[application:get_env(kernel, logger_level)]),erlang:halt()' \
+3&lt; test1.config 4&lt; test2.config</input>
+{ok,error}
+</pre>
```

Comment:
Is there any expected order for configs to be read? Is it `-config` first then `-configfd` entries in declaration order? The base config option can also refer to other config files.

Also what's the merge policy? You mention in the example setting `kernel.logger_level` twice, but what if one of the FDs is a setting of the logger and another one is say, shell history? Are they properly merged within each app definition, or are the setting per-app?

Reply:
> Is there any expected order for configs to be read? Is it -config first then -configfd entries in declaration order?

The order is as declared on the command line. So `-config a -configfd 1 -config b` would read a.config, then fd 1, then b.config. However, `erl -kernel logger_level all` will always override anything given in a config file.

> Also what's the merge policy? You mention in the example setting kernel.logger_level twice, but what if one of the FDs is a setting of the logger and another one is say, shell history? Are they properly merged within each app definition, or are the settings per-app?

The merge policy is per configuration parameter, which is the same as today if you give many `-config`.

Reply:
I have added a description of the merge policy to the documentation.

See:

https://github.com/erlang/otp/pull/2873/commits/ee2920a5daac95f80e2b522db62acb7a3fc36d39#diff-b4ce526e7b5c99616ffad3906c61ed3f879a8290d5728b1926e991ca790e7f8eR43